### ISSUE TITLE
Alves: Footer Credit Migrates To The Middle Of Page

### ISSUE THREAD

**Issue by synora** [Automattic/themes#2976]
## Steps to replicate
1. Create a new page with a blank template. 
2. Add a Group Block
3. Add a 1 Column Layout Grid Block with the following settings: https://d.pr/i/YbpRCD
4. Add 2 blocks within the Layout Grid block: a Cover Block, then a Media And Text Block with content (order of which block is added first doesn't matter).  
Video: https://d.pr/v/08u8yG
5. Publish the page and mark it as your homepage in Design > Customize > Homepage Settings. Choose "Select" for the posts page. 
6. View the homepage.

## Result

The theme's footer credit migrates from the bottom of the page to the second block. 
Video: https://d.pr/v/v7iwnG

When Group and Layout Grid Blocks are not used on a page (so just using a Cover Block and Media/Text Block on the page), the footer credit stays at the bottom of the page. 

## Expected

The footer credit to remain at the bottom of the page. CSS can "fix" this by hiding the credit but that isn't ideal especially on the user's current plan. 

* User report:  #26825302-hc
* Follow-up ticket: #3627711-zd

**Comment by andrewserong** [comment]
This appears to be happening on the Doyle starter design site (and existing sites' homepages based on that design), and when the Doyle design is previewed via wordpress.com/new (the Doyle design uses the Alves theme):

![image](https://user-images.githubusercontent.com/14988353/103835884-8b4be980-50db-11eb-956b-a32afcc8d785.png)

Note that the cover block is also stretched too high vertically on the front end of the site:

![doyle-issue-small](https://user-images.githubusercontent.com/14988353/103836133-3bb9ed80-50dc-11eb-8626-76b8943ee6e0.gif)

Raised in p1609974704240000/1609962160.232600-slack-C013QHLF28Y

**Comment by andrewserong** [comment]
In the interim, we've hidden the Doyle design in Gutenboarding to hide that design for new sites. Once there's a fix for this bug, we can revert https://github.com/Automattic/wp-calypso/pull/48723 and make the design available again.

**Comment by jffng** [comment]
I believe the cover block height issue + migration of the footer credit to the middle of the page should be resolved once the above Gutenberg PR lands on wpcom.

**Comment by jffng** [comment]
The issue is still present since the above Gutenberg PR was reverted. New PR here: https://github.com/WordPress/gutenberg/pull/28455

🤞 hopefully the above can land soon and we can close this.

**Comment by jffng** [comment]
I verified that the underlying cover block height issue was fixed in Gutenberg and now lives on dotcom. @andrewserong you should be able to reactivate Doyle. 

Thanks!

**Comment by andrewserong** [comment]
Thanks @jffng I've opened a revert to reactivate Doyle in https://github.com/Automattic/wp-calypso/pull/49801
